src/components/referenceChecking.vue

```diff
@@ -0,0 +1,48 @@
+<template>
+    <p class="comment_user_text" v-if="isReferenceToUser">
+      <span class="comment_user_text_first_word">@{{ firstWord }}</span>
+      {{ commentText.slice(firstSpaceIndex + 1) }}
+    </p>
+    <p class="comment_user_text" v-else>{{ commentText }}</p>
+  </template>
+  
+  <script>
+  import { mapState } from "vuex";
+  
+  export default {
+    name: "ReferenceChecking",
+    props: {
+      commentText: {
+        type: String,
+        required: true,
+      },
+    },
+    computed: {
+      ...mapState({
+        allAccountsList: (state) => state.allAccounts.allAccountsList,
+      }),
+      firstSpaceIndex() {
+        return this.commentText.indexOf(" ");
+      },
+      firstWord() {
+        if (this.commentText[0] === "@") {
```

Comment:
```suggestion
        if (this.commentText.startsWith("@")) {
```
also what if I type `@@@@username hi man`. then we will also enter this branch and `firstWord` will return `@@@@username` which is not correct